.github/workflows/pr-tests-code.yml

```diff
@@ -0,0 +1,60 @@
+name: Run code tests
+
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+    paths: 
+      - src/**
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Use .NET Core 2.1 LTS SDK
```

Comment:
Why do you need a separate 2.1 SDK installation step if all tests are targeted at 3.1?

Reply:
I forgot; I already fixed up 4.0 so it passes .Net Core 2.1 tests, but not 3.x.  I should take 2.1 testing out of this PR.